cg/store/utils.py

```diff
@@ -7,6 +7,16 @@
 LOG = logging.getLogger(__name__)
 
 
+def case_exists(case_obj, case_id: str):
```

Comment:
I think this function would fit better somewhere under cg/store where we have functionality that relates to the store or as utility function somewhere under cg/cli and then also include the context.abort() since that is what we seem to do when this function returns false

Reply:
I do not understand - It is under cg/store right now!
I could move it to the cg/cli, but the only thing that relates to the cli is the click context and that has really nothing to do with the case_obj having a case_id, except aborting the process. I thought it more general if we do not send in the context as it would require calling this function only when in a click context. But I am happy to move it if you think it is better.

Reply:
yeah, right :)